idd/Energy+.idd.in

```diff
@@ -8841,6 +8841,15 @@ Construction:InternalSource,
        \units m
        \note uniform spacing between tubes or resistance wires in direction
        \note perpendicular to main intended direction of heat transfer
+  N5 , \field Two-Dimensional Position of Interior Temperature Calculation Request
```

Comment:
The word `Request` at the end feels a little odd to me.  If others think it is fine, then go with it, but I feel like it might be OK without `Request`, or with a different word.

Reply:
I can't see the rest of the fields on this object, but just confirm that min-fields doesn't need to be updated here.

Reply:
@Myoldmopar Ok, I'm good with changing this field by removing the word Request.  It's hard to find a short phrase that really captures the intent without being a full sentence.  I guess that is why we have docs?🤷‍♂️

Reply:
Perhaps the master of the IDD @mjwitte has an opinion... It could be that leaving `Request` on there is actually best...

Reply:
Changed this so that the word Request is gone.  I think that the removal of this makes better sense than leaving it.  Will show up in the next commit.

Reply:
I agree with dropping request. An even better field name might be "Two-Dimensional Temperature Calculation Position
The note should mention this field is ignored for 1-d calc?

Other cleanups for this object would be nice:
```
Construction:InternalSource,
       \memo Start with outside layer and work your way to the inside Layer
       \memo Up to 10 layers total, 8 for windows
** This can't be used for window surfaces, can it? So delete ", 8 for windows".
       \memo Enter the material name for each layer
** \min-fields 7
  A1 , \field Name
       \required-field
       \type alpha
       \reference ConstructionNames
  N1 , \field Source Present After Layer Number
       \required-field
       \type integer
       \minimum 1
** \maximum 10
       \note refers to the list of materials which follows
  N2 , \field Temperature Calculation Requested After Layer Number
       \required-field
       \type integer
** \minimum 1
** \maximum 10
       \note refers to the list of materials which follows
  N3 , \field Dimensions for the CTF Calculation
       \required-field
       \type integer
       \minimum 1
       \maximum 2
       \note 1 = 1-dimensional calculation, 2 = 2-dimensional calculation
  N4 , \field Tube Spacing
       \required-field
       \type real
       \units m
** \minimum> 0 ???
       \note uniform spacing between tubes or resistance wires in direction
       \note perpendicular to main intended direction of heat transfer

```

Reply:
@Myoldmopar @mjwitte Ok, I've made the changes that Mike suggested in the name and the other IDD changes (plus one more).  I also clarified the max and min values for each of these in the IO Reference.  That has all been committed to GitHub and I also made another commit merging the latest from develop.